### ISSUE TITLE
Web polling works but fails in android

### ISSUE THREAD

**Issue by sidharth016** [devin-la/ionic-video-chat-v2-issues#75]
Request URL:http://192.168.0.103:5000/socket.io/?EIO=3&transport=polling&t=LonvBA9
Request Method:GET
Status Code:404 Not Found (from disk cache)
On web works for me, on android when i inspect using chrome device inspector, the polling is unable to connect.

**Comment by arzynik** [comment]
1. did you add internet permissions to the android manifest? https://github.com/devin-la/ionic-video-chat-v2-issues/wiki/Building#android
2. are you on the same wifi network as your dev machine?

**Comment by sidharth016** [comment]
Yes did everything, started working on android after changing http:// to ws://  
:/
